aexpect/utils/process.py

```diff
@@ -28,7 +28,7 @@ class CmdError(Exception):
     """Error describing failed command execution"""
 
     def __init__(self, command=None, result=None):
-        super(CmdError, self).__init__()
+        super().__init__()
```

Comment:
Nice but it'd be nice to replace the direct calls used elsewhere as well. Would you like to take care of them now, later or should I do that?

... I'm talking about these:

```diff
diff --git a/aexpect/client.py b/aexpect/client.py
index 4c3665b..9e1f5fc 100644
--- a/aexpect/client.py
+++ b/aexpect/client.py
@@ -501,8 +501,8 @@ class Tail(Spawn):
         self._add_close_hook(Tail._close_log_file)
 
         # Init the superclass
-        Spawn.__init__(self, command, a_id, auto_close, echo, linesep,
-                       pass_fds, encoding)
+        super().__init__(command, a_id, auto_close, echo, linesep,
+                         pass_fds, encoding)
         if thread_name is None:
             self.thread_name = "tail_thread_%s_%s" % (self.a_id,
                                                       str(command)[:10])
```

Reply:
I guess I have missed some, I was scouting for all of them but my IDE seems to have skipped some nevertheless. Sorry about that, will double check and fix this one. In case I miss more don't hesitate to ping me.